script/openqa-clone-custom-git-refspec

```diff
@@ -93,10 +96,11 @@ clone_job() {
     fi
 }
 
-if [[ -z "$host" ]] || [[ -z "$job" ]]; then
-    job_list="${2:?"Need 'job_url' as parameter pointing to the openQA job to clone or 'host' and 'job' variables, e.g. either 'https://openqa.opensuse.org/tests/123456' or 'https://openqa.opensuse.org' and '123456'. Argument can also be a comma-separated list of job URLs or a single host and multiple ids."}"
+if [[ -z "$host" ]] && [[ -z "$job_list" ]]; then
+    echo "Need 'job_url' as parameter pointing to the openQA job to clone or 'host' and 'job' variables, e.g. either 'https://openqa.opensuse.org/tests/123456' or 'https://openqa.opensuse.org' and '123456'. Argument can also be a comma-separated list of job URLs or a single host and multiple ids."
+    exit 1
 fi
 args=${*:3}
-IFS=',' ; for i in $job $job_list; do
-    clone_job "$i"
+IFS=',' ; for i in $job_list; do
+    test -n "$i" && clone_job "$i"
```

Comment:
I think in before we didn't need the test because `for i in $job $job_list` would only include actually present strings, right? I liked the previous approach better :)

Reply:
Before we didn't use an array at all ;-) It seems like the easiest is to use spaces to delimit the list

Reply:
The idea was that either a single job is provided or a comma separated list. When you use spaces for separation then you parse all optional additional arguments into the list as well, I don't think this is a good idea

Reply:
You can

1. Set `job=`
2. Pass a job or several jobs on the CLI
3. Specify a job or several jobs in the PR description

Hence `job_list` is the list of all jobs regardless of where they come from. DRY would suggest not to complicate the loop by being aware of where jobs come from, no?

Reply:
If this doesn't seem to make sense then perhaps we should have another chat about it to avoid more backs and forths and focus on the features. I'd like to add variable support next.

Reply:
Maybe it's easier if we have a PR first with only the github comment thingy. Would this work?

Reply:
No because that is what introduced the conflict. The third source of jobs ;-)